### PR TITLE
Update README.md to have smile

### PR DESCRIPTION
**Pull request by vu22007**

### CODE REVIEW

**Comment by MurrayGroves** [comment]
looks terrible

**Review by MurrayGroves — CHANGES_REQUESTED** [review]
fix this awful code

**Review by a6-webm — CHANGES_REQUESTED** [review]
bruh this is dogass

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # city-fam-2-the-better-one
  yep
  hello
+ featuer :)()()
```
Comment (MurrayGroves) [inline README.md:4]: :((((

**Review by a6-webm — APPROVED** [review]
actually, mb

**Review by MurrayGroves — APPROVED** [review]

**Pull request merged**